src/dune_rules/alias0.ml

```diff
@@ -25,4 +25,5 @@ let check = standard "check"
 let install = standard "install"
 let ocaml_index = standard "ocaml-index"
 let runtest = standard "runtest"
+let pkg_deps = standard "pkg-deps"
```

Comment:
The point of this aliases is to download the sources? If it is, a more suggestive name would be `pkg-sources` or even just `sources`